### ISSUE TITLE
Generating Excel files from HTML content

### ISSUE THREAD

**Issue by aneeshasokan13** [PHPOffice/PhpSpreadsheet#1229]
This is:

```
- [ ] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
Read new html string and add to a new worksheets with name
Add more worksheet and data to the sheet

### What is the current behavior?
It read and write to a single worksheet

### What are the steps to reproduce?

```php
<?php

require __DIR__ . '/vendor/autoload.php';

// Create new Spreadsheet object
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();

$htmlString = '<table>
                  <tr>
                      <td>Hello World</td>
                  </tr>
                  <tr>
                      <td>Hello<br />World</td>
                  </tr>
                  <tr>
                      <td>Hello<br>World</td>
                  </tr>
              </table>';

$reader = new \PhpOffice\PhpSpreadsheet\Reader\Html();
$spreadsheet = $reader->loadFromString($htmlString);

$writer = \PhpOffice\PhpSpreadsheet\IOFactory::createWriter($spreadsheet, 'Xls');
$writer->save('write.xls'); 
```

### Which versions of PhpSpreadsheet and PHP are affected?
Latest versions

**Comment by aneeshasokan13** [comment]
> Unclear what you actually expect to happen here

I have 2 html strings and write these strings to two separate worksheets in a single excel file
Eg.  $string1 = '<table>
                  <tr>
                      <td>Hello World</td>
                  </tr>
                  <tr>
                      <td>Hello<br />World</td>
                  </tr>
                  <tr>
                      <td>Hello<br>World</td>
                  </tr>
              </table>';

$string2= '<table>
                  <tr>
                      <td>Good Morning</td>
                  </tr>                  
              </table>';

I have to write string1 to worksheet with name - Data List 1 and 
string2 to worksheet with name - Data List 2.

**Comment by PowerKiKi** [comment]
Solved via #1212

**Comment by cuongnq6889** [comment]
Fatal error: Call to undefined method PhpOffice\PhpSpreadsheet\Reader\Html::loadFromString()
